### PR TITLE
Updating cAdvisor procedure

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cadvisor-api.md [hunk 1]
```diff
@@ -5,80 +5,103 @@ title: cAdvisor API
...
 
-### Métriques collectées
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
 
-<Tabs groupId="sync">
-<TabItem value="Container-Usage" label="Container-Usage">
+| Alias             | Modèle de service                           | Description                                                            | Découverte |
+|:------------------|:--------------------------------------------|:-----------------------------------------------------------------------|:----------:|
+| Container-Disk-IO | Cloud-cAdvisor-Container-Disk-IO-Api-custom | Contrôle permettant de vérifier les I                                  | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cadvisor-api.md:38]: les I/O?

File: pp/integrations/plugin-packs/procedures/cloud-cadvisor-api.md [hunk 2]
```diff
@@ -5,77 +5,102 @@ title: cAdvisor API
...
-| <container_name_or_id>#container.cpu.utilization.percentage        | CPU utilization                |   %    |
-| <container_name_or_id>#container.cpu.user.utilization.percentage   | CPU User utilization           |   %    |
-| <container_name_or_id>#container.cpu.system.utilization.percentage | CPU System utilization         |   %    |
-| <container_name_or_id>#container.memory.usage.bytes                | Memory usage                   |   B    |
-| <container_name_or_id>#container.memory.cache.usage.bytes          | Cached memory usage            |   B    |
-| <container_name_or_id>#container.memory.rss.usage.bytes            | RSS memory usage               |   B    |
-| <container_name_or_id>#container.swap.usage.bytes                  | SWAP memory usage              |   B    |
+| Service Alias     | Service Template                            | Service Description           | Discovery  |
+|:------------------|:--------------------------------------------|:------------------------------|:----------:|
+| Container-Disk-IO | Cloud-cAdvisor-Container-Disk-IO-Api-custom | Check container disk I        | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cadvisor-api.md:37]: I/O?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cadvisor-api.md (hunk 1 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cadvisor-api.md:38]:
```suggestion
| Container-Disk-IO | Cloud-cAdvisor-Container-Disk-IO-Api-custom | Contrôle permettant de vérifier les I/O disques des containers. | X          |
```

File: pp/integrations/plugin-packs/procedures/cloud-cadvisor-api.md (hunk 2 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-cadvisor-api.md:37]:
```suggestion
| Container-Disk-IO | Cloud-cAdvisor-Container-Disk-IO-Api-custom | Check container disk I/O.    | X          |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**